jenkinsbuilder/weekly-scan.py

```diff
@@ -67,7 +67,14 @@
         desired_tag = entry["desired-tag"]
         email = entry["notify-email"]
 
-        entry_short_name = str(app_id) + "/" + str(job_id)
+        project_name = "{}-{}-{}".format(app_id, job_id, desired_tag)
+
+        try:
+            project = Project.objects.get(name=project_name)
+        except Project.DoesNotExist:
```

Comment:
:+1:  For specific exception handling but there should ideally be a top level exception block as well that can grab anything not handled by specific exception and handle it

Reply:
@mohammedzee1000 : in this case, the possible exception would be the `project_name` not found in the table, and it was the error which was blocking weeklyscan.

Reply:
:+1: